websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({ clientId: "937015924425367643" });
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			details: "Geziniyor",
+			startTimestamp: Math.floor(Date.now() / 1000),
+		},
+		page = document.location.pathname;
```

Comment:
please update the variable name accordingly
```suggestion
		{ pathname }= document.location;
```